### PR TITLE
Add comment about User uniqueness.

### LINKED ISSUE #13: RemoveUserRole does not specify behavior if a group_id, org_id, or user_type are empty
Is the org_id from the hierarchy assumed if empty?
If group_id is empty, is the username removed from all groups?  If yes, is  user_type matched?  if not, what is the point of user_type in RemoveUserRole?

### PR DESCRIPTION
**Pull request by haussli**
Partially fixes #13

### CODE REVIEW

File: ovgs.proto [hunk 1]
```diff
@@ -346,12 +347,11 @@ service OwnershipVoucherService {
...
 
-  // GetOwnershipVoucher issues ownership voucher for the serial number (if it
-  // exists/if applicable)
+  // GetOwnershipVoucher issues ownership voucher and returns TPM public 
+  // key for the serial number (if it exists/if applicable)
   // Errors will be returned:
-  // INVALID_ARGUMENT if serial number, IEN, cert_id, or lifetime are empty,
-  // lifetime is in the past or the IEN supplied isn't applicable for the voucher
-  // issuer
+  // INVALID_ARGUMENT if serial number, IEN or lifetime is empty, lifetime is in the past
```
Comment (harshitk-arista) [inline ovgs.proto:353]: Why the removal of cert_id?

File: ovgs.proto [hunk 2]
```diff
@@ -302,7 +303,7 @@ service OwnershipVoucherService {
   // AddSerial assigns the serial to the group.
   // Errors will be returned:
   // INVALID_ARGUMENT if the serial number is empty or the IEN supplied
-  // isn't applicable for the voucher issuer
+  // isn't applicable f the voucher issuer
```
Comment (harshitk-arista) [inline ovgs.proto:306]: Nit: Typo. f -> for

File: ovgs.proto (hunk 1 above)
Comment (haussli) [reply ovgs.proto]: My mistake.  Blind from staring at it too long.  "all fields are required".

**Comment by harshitk-arista** [comment]
@haussli Resolved the conflicts. Could you please check if nothing was lost? If all looks good, we can merge this.

**Comment by haussli** [comment]
looks complete.  Thanks

**Pull request merged**